src/main/resources/theme/groups/account/messages/messages_en.properties

```diff
@@ -3,3 +3,6 @@ groupsManagementIntroMessage=Manage everything related to advanced Group managem
 showgroupsActivitySidebarTitle=Show My Groups
 joingroupsActivitySidebarTitle=Join group(s)
 enrollmentAccountsSidebarTitle=View enrollment progress
+groupsSidebarTitle=Groups
+groupLabel=My Groups
+groupsIntroMessage=Manage everything related to advanced Group management (groups, enrollment flow etc)
```

Comment:
groupsIntroMessage=Manage everything related to advanced User Group management (groups, enrollment flow etc)